uefi-test-runner/src/main.rs

```diff
@@ -1,6 +1,7 @@
 #![no_std]
 #![no_main]
 #![feature(abi_efiapi)]
+#![feature(negative_impls)]
```

Comment:
I can't see why this is required. Can you please give some details?

Reply:
Yes, `rustc` said it was required when I was implementing my dummy protocol with `#[derive(Protocol]`

Reply:
Alright, thanks for the pointer. What does @nicholasbishop say? Is this fine?

Reply:
I think this is fine, just a consequence of the way we implement protocols. 

(There's a little bit of commentary on `negative_impl` in https://github.com/rust-osdev/uefi-rs/issues/452; it might be something we want to drop in the future. But for this PR it's fine.)